README.md

```diff
@@ -49,4 +49,90 @@ This works the same as [a normal link to that heading](../doc1.md#hello-world).
 
 Linking to a heading without needing to know the destination page can be useful if specifying that path is cumbersome, e.g. when the pages have deeply nested paths, are far apart, or are moved around frequently. And the issue is somewhat exacerbated by the fact that [MkDocs supports only *relative* links between pages](https://github.com/mkdocs/mkdocs/issues/1592).
 
-Note that this plugin's behavior is undefined when trying to link to a heading title that appears several times throughout the site. Currently it arbitrarily chooses one of the pages.
+Note that this plugin's behavior is undefined when trying to link to a heading title that appears several times throughout the site. Currently it arbitrarily chooses one of the pages. In such cases, use [Markdown anchors](#markdown-anchors) to add unique aliases to your headings.
+
+### Markdown anchors
+
+The autorefs plugin offers a feature called "Markdown anchors". Such anchors can be added anywhere in a document, and linked to from any other place. The syntax is `[](){#id-of-the-anchor}`. First you must enable the feature:
+
+```yaml
+# mkdocs.yml
+plugins:
+  - search
+  - autorefs
+      scan_anchors: true
+
+markdown_extensions:
+  - attr_list
+```
+
+Then, add an anchor to a document:
+
+```md
+Somewhere in a document.
+
+[](){#foobar-paragraph}
```

Comment:
Would be cool to explain where this peculiar syntax is coming from. And the fact that it's just a thing that works on its own, just that autorefs adds another meaning to it.

Also maybe doubly stress that `attr_list` is mandatory.

Reply:
Could actually just say that the way to enable this feature is to enable `attr_list` and then actually make such an explicit check

Reply:
I don't think we can easily detect if the feature is used, while the extension is not enabled, so does it really bring anything to check if it is enabled or not? We surely do not want to issue warnings to all current autorefs users when we release a new version. 

We could otherwise force-enable the extension ourselves, but I don't see that as good citizenship. Interactions between extensions and plugins are already complicated enough :smile: